### ISSUE TITLE
How to train with yolov3-spp.cfg?

### ISSUE THREAD

**Issue by toplinuxsir** [AlexeyAB/darknet#2935]
How to train with yolov3-spp.cfg?
Should I modified the yolov3-spp.cfg same as yolov3.cfg? such as params classes and filters
and use pre-trained weights for the convolutional layers as yolov3 https://pjreddie.com/media/files/darknet53.conv.74 or others ?
The same question about yolov3-5l.cfg
If I change width and height ,Should I caculate_anchros ?
Thanks !

**Comment by AlexeyAB** [comment]
Yes, you should do for `yolov3-spp.cfg` and `yolov3-5l.cfg`  all the same as for `yolov3.cfg`
